src/signal.rs

```diff
@@ -23,6 +26,7 @@ use std::ptr::{null, null_mut};
 use std::result;
 use std::thread::JoinHandle;
 
+/// The error cases enumeration for signal handling.
 #[derive(Debug)]
 pub enum Error {
     /// Couldn't create a sigset.
```

Comment:
Right, mixed usage. Thanks for pointing out. I refered to nix::sys::signal and it uses c_int. So let's keep c_int either. Does this make sense to you?

Reply:
Apologies for not commenting on this particular topic earlier; I think that I would also go with `i32`, mainly because it's neater. `c_int` is a `libc` specific alias for `i32` anyway, and should this ever change it won't automatically (potentially) break our interfaces.